app/models/request.rb

```diff
@@ -55,7 +55,7 @@ def paging_origin_library
     return origin unless bib_data.is_a? Folio::Instance # TODO: remove this line post Folio migration
 
     # items are already limited to a single permanent location, so we can just grab the first one
-    bib_data.items.first.permanent_location.details['pagingSchedule'] || origin
+    bib_data.items.first&.permanent_location&.details&.dig('pagingSchedule') || origin
```

Comment:
I understand the first `&.`, but is there a situation where the other two are needed? Don't we always have a details object for a permanentLocation?

Reply:
Isn't this just how `&.` works?

Reply:
okay, so it's chaining for when there are no items rather than specifically protecting against those other structures being nil?